implants/imix/src/config.rs

```diff
@@ -100,13 +113,52 @@ impl Default for Config {
                         "failed to parse retry interval constant, defaulting to 5 seconds: {_err}"
                     );
 
-                    5_u64
+                    5
```

Comment:
is there a reason we removed the type hint?

Reply:
Yeah it confused me and I deleted it broke everything and so this is my attempt to confuse less ppl